lib/blob-store/index.js

```diff
@@ -266,7 +266,7 @@ class PretendCorestore {
       )
     } else if (opts.name === 'db') {
       return this.#coreManager.getWriterCore('blobIndex').core
-    } else if (opts.name === 'blobs') {
+    } else if (opts.name.includes('blobs')) {
```

Comment:
this seems like a reasonable change but not sure how this works in practice. wanted to highlight in order to get clarity

Reply:
We don't actually name any cores, so this will be fine. It's just a way of making sure we return the right core when hyperdrive asks for it. For existing hyperdrives it always reads the blob core key from the blob index anyway. In an ideal world the hyperdrive constructor would accept the blob core and the hyperbee core, but this is a necessary workaround for now.